admin/orders.php

```diff
@@ -62,7 +62,7 @@
 if ($oID) {
   $orders = $db->Execute("SELECT orders_id
                           FROM " . TABLE_ORDERS . "
-                          WHERE orders_id = " . (int)$oID);
+                          WHERE orders_id = " . $oID);
```

Comment:
Our practice is to INTENTIONALLY leave the cast in the SQL queries because then when one is visually scanning the code it is obvious that no SQL-injection risk exists. Otherwise one must visually check around to find out where it was last sanitized ... or not.